src/fastertransformer/devices/OpData.h

```diff
@@ -57,12 +57,14 @@ class OpException : public std::exception {
     const char* what() const noexcept override {
         std::stringstream ss;
         ss << "OpException[" << (int32_t)status_.error_type << "]: " << status_.error_message;
-        return ss.str().c_str();
+        s_ = ss.str();
```

Comment:
why change this ? is it a compiler-related behaviour?

Reply:
The .str().c_str() is a dangling pointer since ss and ss.str() are both destroyed by the end of the function.

Reply:
oh well yes you are right, thanks for pointing out this.